app/templates/collection_history.html.jinja

```diff
@@ -0,0 +1,187 @@
+{% block publication_history_content %}
+  <div class="row gx-3 bg-white p-2 shadow-sm rounded" id="history-page-content">
+    <div class="col-md-8 order-md-1 col-12 mb-md-0 mb-3" id="form-container">
+      <form id="form-element"  class="mb-4">
+        <div class="row">
+          <div class="col-md-8">
+            <h5>{% trans %}entity_type_select_label{% endtrans %} <i
```

Comment:
Lighthouse complains : Heading elements are not in a sequentially-descending order